### DISCUSSION TITLE
Publishing on a free hosting server

### DISCUSSION THREAD

**Discussion by Marii90** [community/community#33617] [Code Security]
I am about to publish my first website it is just html, css and js. I will publish it on a free hosting server(eg netlify ) can someone steal my js source code from developer toll after it is being published? cause my site is very simple doesn't contain server side code. I worked so hard on the js code i don't want it to get stolen. I already uploaded my code on github private repository to get published.

**Comment by Vastyen** [comment]
Hi Marii90

**Your answer it's here https://choosealicense.com**

If you check my repository **(please let me a star)** https://github.com/vastien/be-productive my software license is GPL-2.0. 

The license it's up to you.

**Comment by Marii90** [comment]
@vastien What on earth are u suggesting u suggest that i put an open source licence to my webpage ?!! Did u read and understood my question ?!!

**Comment by Vastyen** [comment]
**The arrogant have no help.**
Good luck
